packages/stark-ui/src/modules/date-picker/date-picker.module.ts

```diff
@@ -1,14 +1,25 @@
-import { NgModule } from "@angular/core";
 import { CommonModule } from "@angular/common";
+import { NgModule } from "@angular/core";
+import { FormsModule, ReactiveFormsModule } from "@angular/forms";
 import { MatDatepickerModule } from "@angular/material/datepicker";
 import { MatInputModule } from "@angular/material/input";
 import { MatMomentDateModule } from "@angular/material-moment-adapter";
 import { StarkDatePickerComponent } from "./components";
 import { TranslateModule } from "@ngx-translate/core";
+import { StarkInputMaskDirectivesModule } from "../input-mask-directives";
 
 @NgModule({
 	declarations: [StarkDatePickerComponent],
-	imports: [CommonModule, MatDatepickerModule, MatInputModule, MatMomentDateModule, TranslateModule],
+	imports: [
+		CommonModule,
+		MatDatepickerModule,
+		StarkInputMaskDirectivesModule,
+		MatInputModule,
+		MatMomentDateModule,
```

Comment:
@SuperITMan according to the [DatePicker docs](https://material.angular.io/components/datepicker/overview#customizing-the-parse-and-display-formats), the `MatMomentDateModule` includes some default formats. Maybe we should use the `MomentDateModule` instead because the formats will be provided by the developer either using our own `STARK_DATE_FORMATS` or their own. What do you think?